Source/Dafny/ConcreteSyntax/LineSegment.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.Dafny
+{
+    
+    class LineSegment : ICanRender
+    {
+        private readonly string _value;
+
+        public LineSegment(string value)
+        {
+            this._value = value;
+        }
+
+        public void Render(TextWriter writer, int indentation, WriterState writerState,
+            Queue<FileSyntax> files)
```

Comment:
Feel free to make the margins wider. It's a safe assumption that project members have screens that comfortably display at least 150 characters.